py_trees_ros/trees.py

```diff
@@ -358,7 +371,7 @@ def setup(
                 read_only=False,
                 floating_point_range=[rcl_interfaces_msgs.FloatingPointRange(
                     from_value=0.0,
-                    to_value=py_trees.common.Duration.INFINITE.value)]
+                    to_value=sys.float_info.max)]
```

Comment:
@stonier This is an unrelated bug fix I found when upgrading to Humble, which will crash py_trees_ros.

Basically, there is a validator in `rcl_interfaces_msgs.FloatingPointRange` that breaks when using `math.int`.

I can put up a separate PR for this alone if you prefer.

Reply:
Aye, sounds like a breaking change across distro boundaries whereas the PR for the node isn't so that could ostensibly get easily back-ported to foxy if someone wanted it there.

Reply:
Created #184 to explore this one too - would like to avoid the case of having an abstraction for this notion (i.e. `py_trees.common.DURATION.INFINITE` that doesn't actually work everywhere. That's confusing.

Reply:
Should be resolved by https://github.com/splintered-reality/py_trees/pull/366.